config_files/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+**IMPORTANT: please be descriptive and provide all required information. Make sure you provide all the information needed for others to review your PR. On sections marked “if applicable”, if that section is not applicable make sure to delete it. Additionally, if your PR closes any open GitHub issues, make sure you include _Closes #XXXX_ in your comment or use the option on the PR's sidebar to add related issues to auto-close the issue that your PR fixes.**
+
+**What is this PR:**
+
+- [ ] Bug fix
+- [ ] Feature
+- [ ] Chore
+
+**Description:**
+
+Please include a summary of the change and which issue is fixed or which feature is introduced. If changes to the behaviour are made, clearly describe what changes.
+
+**Screenshots:**
+
+If changes to the UI are made, please include screenshots of the before and after. 
+
+**Related story:**
+
+Link to the relevant story in Pivotal Tracker.
```

Comment:
Here we could say `relevant story in Pivotal Tracker or issue in GitHub`

Reply:
@arielj the title should also be something like: `Related story/Github Issue:`?

Reply:
Yes sure, to make it more generic.